services/websocket-service/src/notifications/notify.controller.ts

```diff
@@ -1,13 +1,28 @@
-import { Controller, Post } from '@nestjs/common';
+import { Controller, Post, Query } from '@nestjs/common';
 import { AppGateway } from '@/app.gateway';
+import { IsDefined, IsEnum, IsNotEmpty, IsString } from 'class-validator';
+
+enum NotificationType {
```

Comment:
Hey @TzlilSwimmer123 , prefer `as const` objects to enums. See https://www.youtube.com/watch?v=jjMbPt_H3RQ